README.md

```diff
@@ -822,9 +822,11 @@ _The `--fix` option on the command line automatically fixes problems reported by
 
 Enforces consistent use of trailing commas in Object and Tuple annotations.
 
-This rule takes two arguments which both mirror ESLint's default `comma-dangle` rule.
-The first argument is for Object and Tuple annotations.
-The second argument is used for Interface annotations as ESLint's default `comma-dangle` doesn't apply to interfaces - this defaults to whatever the first argument is.
+This rule takes three arguments where the possible values are the same as ESLint's default `comma-dangle` rule:
```

Comment:
These changes need to be made in `.README/*` files and then run `npm run create-readme`.

Reply:
That command is giving me an error:

```
Unhandled rejection Error: Branch ("inexact-object-delimiter-dangle") definition does not exist in the config.
    at Object.gitinfo.getRemoteUrl (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\gitinfo\dist\gitinfo.js:101:13)
    at Object.gitinfo.getUsername (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\gitinfo\dist\gitinfo.js:133:53)
    at Object.gitinfo.getGithubUrl (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\gitinfo\dist\gitinfo.js:51:44)
    at Object.helper.compile (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\gitdown\dist\helpers\gitinfo.js:36:41)
    at Object.gitdown.resolveURLs (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\gitdown\dist\gitdown.js:114:35)
    at C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\gitdown\dist\gitdown.js:54:22
    at tryCatcher (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromise0 (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\bluebird\js\release\promise.js:649:10)
    at Promise._settlePromises (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\bluebird\js\release\promise.js:729:18)
    at _drainQueueStep (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\bluebird\js\release\async.js:93:12)
    at _drainQueue (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\bluebird\js\release\async.js:102:5)
    at Immediate.Async.drainQueues [as _onImmediate] (C:\Users\nwolt\Documents\GitHub Projects\Forks\eslint-plugin-flowtype\node_modules\bluebird\js\release\async.js:15:14)
    at processImmediate (internal/timers.js:439:21)
```

I don't understand why it cares about my git branch. Here's my `git branch` output:

```
> git branch
* inexact-object-delimiter-dangle
  master
```

It looks like `gitinfo` is what's throwing. I added a log statement for [this object](https://github.com/gajus/gitinfo/blob/f0b3c76c6372143e8d538de943f983b5a989b039/src/gitinfo.js#L87) and here's what it shows:

```js
{
  core: {
    repositoryformatversion: '0',
    filemode: false,
    bare: false,
    logallrefupdates: true,
    symlinks: false,
    ignorecase: true
  },
  'remote "origin"': {
    url: 'https://github.com/nwoltman/eslint-plugin-flowtype.git',
    fetch: '+refs/heads/*:refs/remotes/origin/*'
  },
  'branch "master"': { remote: 'origin', merge: 'refs/heads/master' }
}
```

Which makes sense because this is my `.git/config` file:

```
[core]
	repositoryformatversion = 0
	filemode = false
	bare = false
	logallrefupdates = true
	symlinks = false
	ignorecase = true
[remote "origin"]
	url = https://github.com/nwoltman/eslint-plugin-flowtype.git
	fetch = +refs/heads/*:refs/remotes/origin/*
[branch "master"]
	remote = origin
	merge = refs/heads/master
```

And the branch is in my `.git/HEAD` file:

```
ref: refs/heads/inexact-object-delimiter-dangle
```

Reply:
I just hacked into `gitinfo` and made it think I was on the `master` branch. Then that command worked.